docs/learning/useful-resources/setup-new-github-repository.md

```diff
@@ -0,0 +1,59 @@
+# Setup new GitHub repository
+
+Everyone at Thoth Tech has permission to create new GitHub repositories. To ensure that we adhere to
+the
+[Git Contribution guide](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/git-contribution-guide.md),
+we suggest to implement the following GitHub settings.
```

Comment:
Are we assuming that the users know how to find the settings?

Could we add an overview of all the settings? (Almost like a TOC) Therefore, please who know where and what they are doing do not need to scroll and read everything>
Eg.
- delete merged branches automatically: enabled
- access control: role-based
- role assignment: least privilege
- branch protection: enabled

Reply:
Thanks @talzid 

> Are we assuming that the users know how to find the settings?

No, we don't. The steps to navigate are laid out in the body of each section. An example URL path is also included `https://github.com/thoth-tech/handbook/settings`. Do you think we should clarify more?

> Could we add an overview of all the settings?

Yes, we can certainly do that.

Reply:
> Do you think we should clarify more?
I think it might be worthwhile explaining how to access the settings from the repo page, solely as there a few different settings buttons throughout GitHub. Laying it out clearly with one image might help?
Although I am open to rejecting that idea - maybe I am just assuming the worst of our users 🤔 

Then in the following steps you could utilise the menu bar from settings?

Reply:
I have incorporated your suggestions in https://github.com/thoth-tech/handbook/pull/94/commits/6f133e3c69c3c2137a1497cc66003c35e2bd9156. Please let me know what you think.

Reply:
Looks good to me. Thank you!